lib/arclight/engine.rb

```diff
@@ -46,5 +46,9 @@ class Engine < ::Rails::Engine
     initializer 'arclight.views' do
       Blacklight::Configuration.default_values[:view].hierarchy
     end
+
+    initializer 'arclight.assets.precompile' do |app|
+      app.config.assets.precompile += %w[blacklight/ead.svg blacklight/pdf.svg] if defined? Sprockets
```

Comment:
When using the `blacklight_icon` method, do we still need to do this? We haven't needed to for other added icons. Since `Blacklight::Icon` reads the file directly from source, I don't think it needs to touch the asset pipeline.

Reply:
I am finding that errors are raised for me locally when I attempt to remove this, but I will test this on Travis CI.